### PR TITLE
align two fields in new project wizard

### PR DESCRIPTION
**Pull request by elharo**
@briandealwis @hray2003 I'm not 100% sure this one's worth it, since I haven't figured out how to line up with the widgets we inherit from WizardNewProjectCreationPage, but it maybe looks a little better:

image

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java [hunk 1]
```diff
@@ -79,37 +78,42 @@ public void createControl(Composite parent) {
...
+    composite.setLayout(layout);
+    
+    createPackageField(composite, layout, pageValidator);
+    createServiceField(composite, layout, pageValidator);
+  }
 
-    Label packageNameLabel = new Label(composite, SWT.LEAD);
+  // Java package name
+  private void createPackageField(
+      Composite parent, GridLayout layout, ModifyListener pageValidator) {
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java]: Passing the layout is unconventional. And it's not used.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java [hunk 2]
```diff
@@ -79,37 +78,42 @@ public void createControl(Composite parent) {
...
-    GridLayoutFactory.swtDefaults().numColumns(2).applyTo(composite);
   }
 
   // App Engine service name
-  private void createServiceField(Composite container, ModifyListener pageValidator) {
-    Composite composite = new Composite(container, SWT.NONE);
-    // assumed that container has a single-column GridLayout
-    GridDataFactory.fillDefaults().applyTo(composite);
+  private void createServiceField(
+      Composite parent, GridLayout layout, ModifyListener pageValidator) {
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java]: Ditto about passing layout.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties [hunk 3]
```diff
@@ -11,3 +11,4 @@ enter.project.name=Enter a project name.
 project.location.exists: Project location already exists: {0}
 package.ends.with.period: {0} ends with a period.
 package.contains.whitespace: "{0}" contains whitespace.
+app.engine.service="App Engine service:"
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties]: No quotes

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java [hunk 4]
```diff
@@ -79,37 +78,42 @@ public void createControl(Composite parent) {
     Dialog.applyDialogFont(container);
   }
 
-  // Java package name
-  // todo should we turn Label + TextField into a widget of its own? 
-  private void createPackageField(Composite container, ModifyListener pageValidator) {
+  private void createCustomFields(Composite container, ModifyListener pageValidator) {
     Composite composite = new Composite(container, SWT.NONE);
     // assumed that container has a single-column GridLayout
     GridDataFactory.fillDefaults().applyTo(composite);
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties (hunk 3 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties]: done

**Pull request merged**